StripeFinancialConnections/StripeFinancialConnections/Source/API Bindings/FinancialConnectionsAPIClient.swift

```diff
@@ -915,6 +921,27 @@ extension FinancialConnectionsAPIClient: FinancialConnectionsAPI {
             "locale": Locale.current.toLanguageTag(),
             "consent_action": "entered_phone_number_clicked_save_to_link",
         ]
+
+        if let amount, let currency {
```

Comment:
Do we test somehow that we are sending the right params?

Reply:
I'll try think of how we could test this. It'll be a little tricky to test that the values from the `ElementsSessionContext` passed in the `FinancialConnectionsSheet` will be used in the signup API params. 

Couple reasons why: the Financial Connections SDK could use more dependency injection (in this case, we would want to inject an instance of `FinancialConnectionsAPI` to the sheet), and there's also an issue that most of the FC types aren't `public` (or even `@_SPI(STP) public`), which makes DI here tough to implement.

There's definitely a broader action item here, which is to make the FC SDK more testable. I'll track that here [BANKCON-15481](https://jira.corp.stripe.com/browse/BANKCON-15481) so that it doesn't block this work.